src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1445,7 +1445,7 @@ public BundleContext getBundleContext()
      * {@inheritDoc}
      */
     @Override
-    public DtlsControlImpl getDtlsControl(Channel channel)
+    public DtlsControlImpl getSrtpControl(Channel channel)
```

Comment:
No, I don't see anything outside IceUdpTransportManager which uses the return value as DtlsControl.